spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/spark/streaming/SparkStreamingPlugin.java

```diff
@@ -107,8 +107,8 @@
 	private Map<Module, JavaStreamingContext> streamingContexts = new HashMap<>();
 
 	@Autowired
-	public SparkStreamingPlugin(MessageBus messageBus, ZooKeeperConnection zkConnection) {
-		super(messageBus, zkConnection);
+	public SparkStreamingPlugin(MessageBus messageBus) {
+		super(messageBus, null);
```

Comment:
I think there is a 1 parameter version of this constructor that you can use instead of passing `null`.